example/testapp/apitest/features/basic.feature

```diff
@@ -75,9 +77,6 @@ Feature: Basic Features for lodash-match-pattern
         }
       ]
       """
-
-  Scenario: Users are searched correctly
-    Given the database is populated with sample users
     When the client gets "/users?name=alec"
```

Comment:
Actually we found out recently that one should never program GET requests that modify any server state.  This is because some browsers aggressively preload every URL they find.